tests/test_sqlite.py

```diff
@@ -56,6 +56,6 @@
     "django.core.context_processors.static",
     "django.core.context_processors.media",
     "django.core.context_processors.request",
-    "mezzanine.conf.context_processors.settings",
-    "mezzanine.pages.context_processors.page",
+    #"mezzanine.conf.context_processors.settings",
+    #"mezzanine.pages.context_processors.page",
```

Comment:
Commenting code is often a bad idea (remove it or leave it). But here, I'm not sure. Could you clarify the reason behind it?